pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge PDF:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
+
+- Separate PDF into individual pages:
+
+`pdftk {{input.pdf}} burst`
+
+- Rotate third page by 90 degrees clockwise:
```

Comment:
@waldyrious It would not work with a space. As far as I've understood, detailed syntax usage is not given in the tldr pages and the user should refer to the man page for more information, right ?